src/Metal/MTLDevice.cs

```diff
@@ -42,6 +42,28 @@ public static partial class MTLDevice {
 				return system_default;
 			}
 		}
+
+#if MONOMAC
+		//[Mac (10,13, onlyOn64: true), NoiOS, NoWatch, NoTV]
+		//[DllImport (Constants.MetalLibrary)]
+		//static extern IMTLDevice[] MTLCopyAllDevicesWithObserver ([NullAllowed] out NSObject observer, MTLDeviceNotificationHandler handler);
```

Comment:
Are you sure it's not XM Classic that's failing to build? Can you gist the entire build log?